src/client/fontengine.cpp

```diff
@@ -56,25 +52,17 @@ FontEngine::FontEngine(gui::IGUIEnvironment* env) :
 
 	readSettings();
 
-	if (m_currentMode != FM_Simple) {
-		g_settings->registerChangedCallback("font_size", font_setting_changed, NULL);
-		g_settings->registerChangedCallback("font_bold", font_setting_changed, NULL);
-		g_settings->registerChangedCallback("font_italic", font_setting_changed, NULL);
-		g_settings->registerChangedCallback("font_path", font_setting_changed, NULL);
-		g_settings->registerChangedCallback("font_path_bold", font_setting_changed, NULL);
-		g_settings->registerChangedCallback("font_path_italic", font_setting_changed, NULL);
-		g_settings->registerChangedCallback("font_path_bolditalic", font_setting_changed, NULL);
-		g_settings->registerChangedCallback("font_shadow", font_setting_changed, NULL);
-		g_settings->registerChangedCallback("font_shadow_alpha", font_setting_changed, NULL);
-		g_settings->registerChangedCallback("font_size_divisible_by", font_setting_changed, NULL);
-		g_settings->registerChangedCallback("fallback_font_path", font_setting_changed, NULL);
-	}
+	const char *settings[] = {
+		"font_size", "font_bold", "font_italic", "font_shadow", "font_shadow_alpha",
+		"font_size_divisible_by",
+		"font_path", "font_path_bold", "font_path_italic", "font_path_bolditalic",
+		"fallback_font_path",
+		"mono_font_path", "mono_font_size", "mono_font_size_divisible_by",
+		"screen_dpi", "gui_scaling",
+	};
```

Comment:
`font_path_bolditalic` -> `font_path_bold_italic`
Add `mono_font_path_bold`, `mono_font_path_italic`, and `mono_font_path_bold_italic`

I thought I had fixed those at some point or another...